fidesctl/src/fidesctl/cli/__init__.py

```diff
@@ -40,7 +42,6 @@
 @click.group(
     context_settings=CONTEXT_SETTINGS,
     invoke_without_command=True,
-    chain=True,
```

Comment:
i did this on purpose, and i think it was to make the auto-generated docs work. can you confirm this doesn't break them?

Reply:
I had heard that somewhere recently and ended up testing it (was looking at this as part of the initial `scan` command) - tested using `make docs-serve` to generate the docs locally, is that the best way to confirm @ThomasLaPiana ?

Reply:
Per earlier conversation @ThomasLaPiana - I _think_ the auto-generated docs are still working looking at this:
![image](https://user-images.githubusercontent.com/6758661/150195433-395b4d51-cf67-406f-b695-2aaba2220cd9.png)

Reply:
sounds good, i'll take a look and double check but thats good news!